web-ng/src/app/components/form-field-editor/form-field-editor.component.scss

```diff
@@ -51,6 +46,22 @@
   margin-right: 20px;
 }
 
+.question-container {
+  border-radius: 10px;
+  padding: 0px 10px;
+  margin-top: 10px;
+  border: 1px #e6e7ea;
```

Comment:
Please be sure to use Angular themes (`mat-color` etc.) instead of color literals.